README.md

```diff
@@ -58,28 +58,54 @@ following commands in the cloned Git repository:
 
 You can read more about this in the [Git for Windows wiki](https://github.com/git-for-windows/git/wiki/Git-cannot-create-a-file-or-directory-with-a-long-path).
 
-## Setup
-
-To run all of Dependabot Core, you'll need Ruby, Python, PHP, Elixir, Node, Go,
-Elm, and Rust installed. However, if you just wish to run it for a single
-language you can get away with just having that language and Ruby.
-
-While you can run Dependabot Core without Docker, we provide a development
-Dockerfile that bakes in all required dependencies. In most cases this is the
-best way to work with the project.
+## Running with the `dependabot` CLI
+
+Some ecosystems in Dependabot Core have specific system dependencies.
+For example, running the `composer` ecosystem requires
+a particular version of `php` and other libraries to be installed.
+As such, we use containers to build and run Dependabot Core.
+
+The official Docker images are hosted
+on Docker Hub at [`dependabot/dependabot-core`](https://hub.docker.com/r/dependabot/dependabot-core)
+and on GitHub Container Registry at [`ghcr.io/dependabot/dependabot-core`](https://github.com/dependabot/dependabot-core/pkgs/container/dependabot-core).
+
+You can use the [`dependabot` CLI][dependabot-cli]
+to simulate a Dependabot update job for the provided ecosystem and repo.
+To test out the behavior of a development version of `dependabot-core`,
+build `Dockerfile.updater` and
+run the `dependabot update` subcommand
+passing the image tag to the `--updater-image` option.
+
+```console
+$ docker build . -f Dockerfile.updater \
+                 -t dependabot-updater-dev
```

Comment:
Needs `--build-arg OMNIBUS_VERSION=latest` or similar.

It's probably worth adding a step to build `dependabot-core` so that local changes can be reflected, particularly given that's how `docker-dev-shell` worked.